web/pdf_page_view.js

```diff
@@ -1004,22 +978,36 @@ class PDFPageView {
     const canvas = document.createElement("canvas");
     canvas.setAttribute("role", "presentation");
 
-    // Keep the canvas hidden until the first draw callback, or until drawing
-    // is complete when `!this.renderingQueue`, to prevent black flickering.
-    canvas.hidden = true;
```

Comment:
If this is being removed, can we also the following CSS now?
https://github.com/mozilla/pdf.js/blob/a4eb8407c1b3695be0d1f91791f393655b476dd9/web/pdf_viewer.css#L91-L93